maat-court-data-api/src/test/java/gov/uk/courtdata/billing/controller/ApplicantCclfResetControllerTest.java

```diff
@@ -0,0 +1,51 @@
+package gov.uk.courtdata.billing.controller;
+
+import gov.uk.courtdata.billing.service.ApplicantCclfResetService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.bean.override.mockito.MockitoBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.verify;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(ApplicantCclfResetController.class)
+@AutoConfigureMockMvc(addFilters = false)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+class ApplicantCclfResetControllerTest {
```

Comment:
These tests will need to change once we change to pass in a list of ids, so no comments here until then